systems/registry/api-gateway/pkg/rest/router.go

```diff
@@ -298,18 +298,13 @@ func (r *Router) getNetworkHandler(c *gin.Context, req *GetNetworkRequest) (*net
 	return r.clients.Network.GetNetwork(req.NetworkId)
 }
 
-func (r *Router) getNetworksHandler(c *gin.Context, req *GetNetworksRequest) (*netpb.GetByOrgResponse, error) {
-	orgName, ok := c.GetQuery("org")
-	if !ok {
-		return nil, &rest.HttpError{HttpCode: http.StatusBadRequest,
-			Message: "org is a mandatory query parameter"}
-	}
+func (r *Router) getNetworksHandler(c *gin.Context, req *GetNetworksRequest) (*netpb.GetNetworksResponse, error) {
```

Comment:
Yeah, I'm sure now. We can definitely remove   `req *GetNetworksRequest` since we are not using it!